### ISSUE TITLE
incorrect name for fedora package in README.md

### ISSUE THREAD

**Issue by Kn4ughty** [Luwx/Lightly#234]
The name for the fedora package in the readme is incorrect, at least on Asahi Linux.
```sh
$ sudo dnf install Lightly
No match for argument: Lightly
  * Maybe you meant: lightly
Error: Unable to find a match: Lightly
```

**Comment by theaino** [comment]
The package is also kind of broken.

**Comment by TKK13909** [comment]
Hey, this repo has been abandoned. https://github.com/Bali10050/Lightly is being actively maintained so you should probably give it a try.

**Comment by Kn4ughty** [comment]
Yeah i think you're right, however this issue isn't applicable to the fork since it doesn't have an official fedora package anyway. Thanks for the heads up anyway

**Comment by TKK13909** [comment]
It does have a copr though. https://copr.fedorainfracloud.org/coprs/deltacopy/lightly-qt6/monitor/
